economy/migrations/0002_auto_20200407_1026.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.7 on 2020-03-29 15:01
```

Comment:
I would suggest naming this e.g. `0002_create_soci_master_account_20200407_1026.py` so it's easier to keep track of it in the future.